code/script.js

```diff
@@ -0,0 +1,95 @@
+const userName = 'JohannaMJ';
+const REPOS_API_URL = `https://api.github.com/users/${userName}/repos`;
+const USER_URL = `https://api.github.com/users/${userName}`;
+const reposContainer = document.getElementById('projects');
+const profileContainer = document.getElementById('profileContainer');
+
+const userProfile = () => {
+  fetch(USER_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      profileContainer.innerHTML = `
+        <div class="icon-container">
+          <a href="${data.blog}" target="blank"><img class="linkedin-icon" src="./assets/icon.png"></a>
```

Comment:
Love that you added a linked in icon with and achor tag! Great idea and it matches your styling!